docs(README): fix grammatical errors (#552)
invertase/notifee: README.md

| Topic                                                             |                                                                          |
| ----------------------------------------------------------------- | ------------------------------------------------------------------------ |

> | [Appearance](https://notifee.app/react-native/docs/ios/appearance)           | Change now, the notification that is displayed to your users.       |

Reviewer: The original was just a typo of "now" vs "how" I think
```suggestion
| [Appearance](https://notifee.app/react-native/docs/ios/appearance)           | Change how the notification is displayed to your users.       |
```
